Operational-Procedures/Auxiliary-Telescope/LATISS-Master-Calibrations-Procedure/latiss-master-calibrations-procedure.rst

```diff
@@ -0,0 +1,212 @@
+.. |author| replace:: *Andrés A. Plazas Malagón*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. _Latiss-Master-Calibrations-Procedure:
+
+###########################
+Latiss Master Calibrations Generation Procedure
+###########################
+
+.. _Latiss-Master-Calibrations-Procedure-Overview:
+
+Overview
+========
+
+This procedure describes how to call scripts to produce master calibrations for LATISS from the LSST Operations and Visualization Enviroment (LOVE) at the Summit :ref:`operational environment <Observing-Interface-Operational-Environments>`. 
+
+The script will have the option to: 
+
+- command LATISS to take a number of calibrations frames:
+   - biases
+   - darks 
+   - flats
+- call the corresponding Rubin Science Pipelines calibration generation pipetask to produce biases, darks, flats, defects and Photon Transfer Curves (PTCs) via the OCS-Controlled Pipeline System (OCPS),
+- verify the resulting calibration (see the package `cp_verify`_ and `DMTN-101`_),
+- certify the resulting calibration with a given range of validity dates.
+
+The script currently has the option (via the `script_mode` parameter in the configuration options) to:
+
+- take only biases, 
+- take biases and darks, and 
+- take biases, darks, and flats. 
+  
+These options are constrained by the fact that one calibration depends on the existence of the previous one (i.e., to build a dark, a bias is necessary, and to build a flat, a dark and a bias are necessary).
+
+If desired, defects can be constructed from darks and flats (in which case  `script_mode` must be ``BIAS_DARK_FLAT``), and a PTC per detector per amplifier constructed form the flats. Note that the PTC assumes that a sequence of flat pairs has been taken, each pair taken at the same exposure time (in order to produce a PTC, `script_mode` must be ``BIAS_DARK_FLAT`` too).
+
+For more information about calibrations production (including verification and certification), please consult the `Constructing Calibrations documentation`_.
+
+.. _cp_verify: https://github.com/lsst/cp_verify
+.. _DMTN-101: https://dmtn-101.lsst.io/
+.. _Constructing Calibrations documentation: https://lsst.ncsa.illinois.edu/~czw/pipelines_lsst_io/_build/html/modules/lsst.cp.pipe/constructing-calibrations.html 
+
+.. _Latiss-Master-Calibrations-Procedure-Prerequisites:
+
+
+Prerequisites
+=============
+
+- You should be logged into the LSST Operations and Visualization Enviroment (LOVE) at the Summit :ref:`operational environment <Observing-Interface-Operational-Environments>`.
+- The script assumes (and checks) that the ``LATISS`` and ``OCPS`` components are all ``ENABLED``, and that the latter has been ``ENABLED`` with the configuration of ``LATISS``. The instrument and the ``OCPS`` can be enabled, for example, from a notebook: 
```

Comment:
We should probably have a separate procedure to that shows how to enabled LATISS and OCPS, instead of telling users to go to notebook to do it. 

I would suggest you create placeholders for these procedures and link them here, and them remove the code snippet below. 

I think we need 3 new (placehoders for) procedures;

1 - enable LATISS
2 - enable OCPS
3 - prepare ATCS for flat-fields.

Again, you don't need to write these procedures but create empty pages as placeholders and link them here.